client/src/app/components/stardust/NewOutput.tsx

```diff
@@ -0,0 +1,190 @@
+import { BASIC_OUTPUT_TYPE, ALIAS_OUTPUT_TYPE, FOUNDRY_OUTPUT_TYPE, NFT_OUTPUT_TYPE,
+    TREASURY_OUTPUT_TYPE, SIMPLE_TOKEN_SCHEME_TYPE } from "@iota/iota.js-stardust";
+import classNames from "classnames";
+import React, { Component, ReactNode } from "react";
+import { ReactComponent as DropdownIcon } from "./../../../assets/dropdown-arrow.svg";
+import FeatureBlock from "./FeatureBlock";
+import { NewOutputProps } from "./NewOutputProps";
+import { NewOutputState } from "./NewOutputState";
+import UnlockCondition from "./UnlockCondition";
+
+/**
+ * Component which will display an output.
+ */
+class NewOutput extends Component<NewOutputProps, NewOutputState> {
```

Comment:
Why is it called "New" output ? Can we just call it "Output" ?

Reply:
There is already a component in the folder named Output that's why i named it NewOutput. You can suggest a better name and we can call it that.